modules/nw-egress-ips-static.adoc

```diff
@@ -39,10 +39,7 @@ $ oc patch netnamespace project1 --type=merge \
   -p '{"egressIPs": ["192.168.1.100"]}'
 ----
 +
-You can set `egressIPs` to two or more IP addresses on different nodes to
-provide high availability. If multiple egress IP addresses are set, pods use the
-first IP in the list for egress, but if the node hosting that IP address fails,
-pods switch to using the next IP in the list after a short delay.
+You can set `egressIPs` to two or more IP addresses on different nodes to provide high availability. If multiple egress IP addresses are set then pods use all egress IP addresses roughly equally.
```

Comment:
EgressIP load-balance is  not only for manually assignment, but also for auto assignment. 
Under paragraph "considerations-automatic-egress-ips",  "Only a single egress IP address per namespace is supported when using the automatic assignment mode." should be removed.

Reply:
@huiran0826, do I need to change this for automatic assignment as well?

```
. Update the `NetNamespace` object with the egress IP address using the
following JSON:
+
[source,terminal]
----
 $ oc patch netnamespace <project_name> --type=merge -p \ <1>
  '{
    "egressIPs": [
      "<ip_address>" <2>
    ]
  }'
----
<1> Specify the name of the project.
<2> Specify a single egress IP address. Using multiple IP addresses is not
supported.
```

Thanks!

Reply:
@huiran0826, I think the preview is not correct; The text is wrapped in a conditional and will not appear for OCP >= 4.8:

```
ifeval::[{product-version} < 4.8]
- Only a single egress IP address per namespace is supported when using the automatic assignment mode.
endif::[]
```

Reply:
> <2> Specify a single egress IP address. Using multiple IP addresses is not
> supported.

Yes, I think so. "<2> Specify a single egress IP address. Using multiple IP addresses is not supported" needs update.

Reply:
@huiran0826, I updated to:

`modules/nw-egress-ips-automatic.adoc:35`
> `<ip_address>`:: Specifies one or more egress IP addresses.